versions.gradle

```diff
@@ -12,8 +12,8 @@ ext {
     targetSDKVersion = 33
 
     // project versioning
-    versionCode = 4
-    versionName = '1.1.1'
+    versionCode = 6
+    versionName = '1.2.0-beta1'
```

Comment:
if you want to do a beta version, https://semver.org/#spec-item-9 should follow the semantic versioning for it, should be `1.2.0-beta.1`

Reply:
Thanks for finding the semver on this. I should have looked it up but I followed an example from the cocoapods doc. I will leave iOS as is for now and update android to not have the number so `1.2.0-beta` and if we have more patches on the beta related to rich push we can append the number.

![image](https://github.com/klaviyo/klaviyo-android-sdk/assets/118314354/5e3649fd-f7a7-4690-834b-a5293e92ba13)